views/layouts/main.njk

```diff
@@ -11,12 +11,11 @@
     <body{% if bodyClass %} class="{{ bodyClass }}"{% endif %}>
         {% include "../includes/accessibility.njk" %}
 
-        {# main content area #}
-        <main>
+        <div class="global-container">
```

Comment:
This is bit of a semantic fix. This arguably shouldn't be a `main` as

1. The _content_ is really the main area, not the whole page
2. We have `role=main` on all container elements inside page templates so semantically we have two mains

Reply:
Fair enough!